### PR TITLE
Order display voxel sphere

### PR DESCRIPTION
**Pull request by kerautret**
Minor PR to improve OpenGL transparency display to be able to display such cases:
capture d ecran 2015-11-28 a 22 23 06

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -20,6 +20,12 @@
  - Minor improvements of default settings in Viewer3D. (David
    Coeurjolly, [#1066](https://github.com/DGtal-team/DGtal/pull/1066))
 
+ - change the chronological order to diplay primitives (in the draw
+   function) in order to see see the cube primitive through the
```
Comment (dcoeurjo) [inline ChangeLog.md]: "See see"
PrimitiveS

File: ChangeLog.md [hunk 2]
```diff
@@ -20,6 +20,12 @@
...
+   transparency of the ball primitive. (Bertrand Kerautret,
+   [#1081](https://github.com/DGtal-team/DGtal/pull/1081))
```
Comment (dcoeurjo) [inline ChangeLog.md]: Ball primitiveS ?

**Comment by dcoeurjo** [comment]
Otherwise the PR is fine. (Until we would like to see solid balls inside transparent cubes ;))

**Comment by kerautret** [comment]
Thanks for the tipo.
Of course ;) But from the previous refactoring it should be now easy to apply an chronological order or an user manual order. We could discuss on it and I can look it.

**Comment by dcoeurjo** [comment]
ok thanks, merging

**Pull request merged**